test/e2e_test.go

```diff
@@ -79,6 +84,7 @@ const (
 	serverEnv = "REKOR_SERVER"
```

Comment:
Can this be removed? It doesn't look like this env var is read by Cosign.

If we wanted to confirm the verification is offline, you could pass an invalid URL for Rekor to `verify`